tests/tests_suite_test.go

```diff
@@ -44,6 +44,10 @@ var (
 )
 
 func getDir() string {
+	// if inside Docker container, don't return nothin'
```

Comment:
@mboersma talked about this option at one point, not really having a strong preference. Is it a better practice?